src/libraries/Microsoft.Extensions.DependencyModel/src/UnifiedJsonReader.Utf8JsonReader.cs

```diff
@@ -1,158 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-
-using System;
-using System.Collections.Generic;
-using System.Diagnostics;
-using System.Reflection;
-using System.Text.Json;
-
-namespace Microsoft.Extensions.DependencyModel
-{
-    internal ref struct UnifiedJsonReader
```

Comment:
Was this duplicated for Newtonsoft? Which is no longer necessary?

Reply:
Correct. It was an abstraction that was used to be able to switch between Newtonsoft and STJ. See https://github.com/dotnet/core-setup/pull/5009.

We no longer build this library for TFMs that STJ doesn't support. So we don't need this abstraction anymore. We can just use STJ directly.